plugins/file-open-save/core/src/main/javascript/app/components/files/files.component.js

```diff
@@ -183,14 +184,12 @@ define([
       vm.sortState = vm.sortField === field ? vm.sortState : 0;
       switch (vm.sortState) {
```

Comment:
@tgf @bmorrise ...got clarification from UX. Wants same behavior as DET. DET toggles asc/desc after 1st click. We are good. :)